TFT/src/User/Configuration.h

```diff
@@ -1264,6 +1264,14 @@
  */
 #define PROGRESS_BAR_COLOR 0  // Default: 0
 
+/**
+ * The color of the values on the status screen
+ * Some topics require the inverse display of color values on live status screen icons.
+ * Uncomment for example: THEME_Rep Rap Firmware Dark 
+ * Comment for standard color
+ */
+//#define VALUE_COLOR // Default: commented (disabled)
+
```

Comment:
I would put this before PROGRESS_BAR_COLOR or after MARKED_PROGRESS_BAR so progress bar stuff remains together.

Reply:
OK

Reply:
Thx